build/RELEASING.md

```diff
@@ -33,12 +33,12 @@ in `PublicAPI.Shipped.txt` files and cleanup corresponding `PublicAPI.Unshipped.
 on the `main` branch is the one which added/updated the Changelog to
 the project being released. *This latest commit will be tagged on the release.*
 
-1. Create and push git tag for the project and the version of the project
+1. Create and push signed git tag for the project and the version of the project
 you want to release. The version should be the one used in the **Pre-steps** to
 update the Changelog.
 
     ```powershell
-    git tag -a <PROJECT TAG PREFIX>-<VERSION> -m "<PROJECT TAG PREFIX>-<VERSION>"
+    git tag -s -a <PROJECT TAG PREFIX>-<VERSION> -m "<PROJECT TAG PREFIX>-<VERSION>"
```

Comment:
will ask all maintainers to test and confirm before merge.

Reply:
This will require maintainers to create and manage their own GPG key. I'd be fine with this as I've been doing it for years, but most people I work with have not adopted this practice. Do other maintainers want to do this?

https://docs.github.com/en/authentication/managing-commit-signature-verification/generating-a-new-gpg-key

Reply:
> This will require maintainers to create and manage their own GPG key.

This is fine with me as well.

Reply:
@CodeBlanch, @cijothomas, what do you think?